anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
Ah, that's not it.

My change doesn't prevent that.  Say you have a note with {{c1::foo [sound: foo]}}  and {{c2::bar [sound:bar]}}  then later you have {{c3:: foobar[sound:foobar]}}  and foobar again for the same card {{c3::foobar[sound:foobar]}}  and then just for kicks you have another word on card 3  {{c3::some other words[sound:some other words]}}

What my fix does is allow you to only hear the sounds for the clozes on the card you are using.  so only card 1 will have sounds from the c1 clozes.   or card 3 will have 3 sounds because there were three c3 clozes.

It also doesn't preclude putting sounds outside of the clozes.

Currently if I made a note with the clozes above,  Everycard would play every sound when it came up and then play them again when I got the answer, regardless of which cloze I was looking at.